app/views/completed_transaction/show.html.erb

```diff
@@ -38,47 +38,50 @@
     </div>
   <% end %>
 
-  <h2 class="satisfaction-survey-heading">Satisfaction survey</h2>
+  <% if show_survey? %>
+    <h2 class="satisfaction-survey-heading">Satisfaction survey</h2>
```

Comment:
*cough* https://github.com/alphagov/frontend/pull/1035 ;)  If you can leave this as-is it might make my eventual rebase merge nightmare easier.